core/trino-main/src/main/java/io/trino/operator/StreamingDirectExchangeBuffer.java

```diff
@@ -32,6 +32,7 @@
 
```

Comment:
Change commit message to "Avoid unblocking more ExchangeOperators than necessary in exchange buffer"